### ISSUE TITLE
Feature Request:  provide more options in how rdf:type values are handled

### ISSUE THREAD

**Issue by mdw00d** [neo4j-labs/neosemantics#143]
It would be helpful to have a parameter "handleTypeUris" to importRDF/importRDFSnippet that could be assigned to one of the four values  'LABEL','EDGE','BOTH','PROPERTY' and had the following semantics:

value | Description
-------|------------
'LABEL' | type becomes a label, where the label is generated in keeping with handleVocabUris
'EDGE' | type is used to generate an edge between the node and the node with the specified URI
'BOTH' | both a label is generated as well as an edge	
'PROPERTY' | the type becomes an attribute of property named ‘type’; if there are multiple types for a node, then they are handled in the manner specified by handleMultiVal

Currently there's a parameter typesToLabels which if true causes the rdf:type value to be attached as a label; otherwise, it is attached as an edge.

In order to do inferencing against an OWL-based ontology, it is helpful if not necessary to have the rdf:type values represented as edges.  However, it is also very convenient to have nodes labeled with their type; I assume that this results in more efficient lookups and graph operations than having to always traverse an edge to a type node.

**Comment by giantamoeba** [comment]
I too need the 'BOTH' option, currently I set `typesToLabels` to `false` and then run the following query to add them as labels:
```
      MATCH 
       (n)-[:rdf__type]->(c)
      WHERE NOT ('rdf__' + c.id) in labels(n)
      CALL apoc.create.addLabels(n, ['rdf__' + c.id])
```

**Comment by jbarrasa** [comment]
This has been addressed in 4.0.

The `typesToLabels` param has been replaced with `handleRDFTypes` with the following valid values: 
* 'LABELS', rdf:type statements are imported as node labels in Neo4j
* 'NODES', rdf:type statements are imported as rdf__type relationships to a node representing the class
* 'LABELS_AND_NODES' rdf:type statements are stored (in a redundant way) as in both previously described cases.

Link to [manual](https://neo4j.com/docs/labs/nsmntx/current/reference/#_graph_config_params_global_settings).

Hopefully this will give more flexibility. 

Not convinced the `PROPERTY` option added much value having the `LABEL` one, so we've not implemented it for now.  

Feel free to close the issue if this satisfies your use case or comment on why `PROPERTY` should be added.

JB.

**Comment by mdw00d** [comment]
Thanks for adding this feature.

I may have suggested the 'PROPERTY' option for completeness; what you've implemented I think covers what we need.

Thanks again.
